### PR TITLE
[CPU] zero-point fix

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *CPU zero-point fuse fix*

### Tickets:
 - *CVS-111175*

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/include/conv_with_zero_point_fuse.hpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using commonConvParams =  std::tuple<
+    InferenceEngine::SizeVector,    // Kernel size
+    InferenceEngine::SizeVector,    // Strides
+    std::vector<ptrdiff_t>,         // Pad begin
+    std::vector<ptrdiff_t>,         // Pad end
+    InferenceEngine::SizeVector,    // Dilation
+    size_t,                         // Num out channels
+    ngraph::op::PadType,            // Padding type
+    size_t                          // Number of groups
+>;
```
Comment (maxnick) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/include/conv_with_zero_point_fuse.hpp:29]: Do we really need all these parameters for the fusing test?
Can't we simply hardcode them?

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp
Comment (maxnick) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp:1]: Unusually we write a short description about the purpose of the test (often with a scheme).

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/include/conv_with_zero_point_fuse.hpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/subgraph_tests/include/conv_with_zero_point_fuse.hpp]: done, thanks

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp:1]: was added

File: src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp [hunk 2]
```diff
@@ -432,7 +433,7 @@ std::vector<CPUSpecificParams> filterCPUInfoForDevice(const std::vector<CPUSpeci
 
         if (selectedTypeStr.find("jit") != std::string::npos && !InferenceEngine::with_cpu_x86_sse42())
             continue;
-        if (selectedTypeStr.find("sse42") != std::string::npos && !InferenceEngine::with_cpu_x86_sse42())
+        if (selectedTypeStr.find("sse42") != std::string::npos && (!InferenceEngine::with_cpu_x86_sse42() || int8SupportOnly))
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp]: removed, thanks

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+namespace SubgraphTestsDefinitions {
+
+std::string ConvWithZeroPointFuseSubgraphTest::getTestCaseName(testing::TestParamInfo<convConcatCPUParams> obj) {
+    std::ostringstream result;
+    nodeType type;
+    SizeVector inputShapes;
+    std::tie(type, inputShapes) = obj.param;
+
+    result << "Type=" << nodeType2str(type) << "_";
```
Comment (maxnick) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_with_zero_point_fuse.cpp:20]: May be it does make sense to print the shape as well?

**Review by maxnick — APPROVED** [review]
There is one minor comment is left.

**Pull request merged**